config/locales/pt-BR.yml

```diff
@@ -0,0 +1,175 @@
+pt-BR:
+  title: "RubyGems.org"
+  subtitle: "o host de gems da sua comunidade"
+  help_url: "http://help.rubygems.org"
+  please_sign_up: "Accesso Negado. Por favor se registre em: https://rubygems.org"
```

Comment:
wow.. that's weird. I could swear I saw it in the english version.
That's alright then. I'll change it.